mods/cnc/rules/world.yaml

```diff
@@ -244,7 +244,7 @@ World:
 		BaseActor: mcv
 		SupportActors: e1,e1,e1,e1,e1,e2,e2,e2,e3,e3,apc,mtnk
 	SpawnMPUnits:
-		StartingUnitsClass: light
+		StartingUnitsClass: none
```

Comment:
This is already the default, so you should be able to just delete the line

Reply:
Fixed